### PR TITLE
Update README instructions, GSoC Report

### PR DESCRIPTION
**Pull request by AvishrantsSh**
Signed-off-by: Avishrant Sharma <avishrants@gmail.com>

### CODE REVIEW

**Comment by AvishrantsSh** [comment]
@tdruez, can you take a look at the README?

File: docs/source/contribute/GSoC'21 Final Report.rst [hunk 1]
```diff
@@ -11,111 +11,147 @@ Project Title: Integration of Alternate Code Analysis Tools
...
+
+- Further, license key mapping between License Classifier and ScanCode.io was generated and
+  integrated into the package. It helps in maintaining consistency between the two.
+
+- This Python package is available at `GoLicense-Classifier`_
 
 Developing a pipeline to run license detection
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- After developing the package, it was necessary to develop a pipeline that can be used to
+  establish communication with ScanCode.io.
```
Comment (tdruez) [inline docs/source/contribute/GSoC'21 Final Report.rst]: > establish communication with ScanCode.io
Is not great in that context.

I would suggest something along:
"After developing the package, it was necessary to integrate it into a pipeline that can be used to execute the License Classifier from the ScanCode.io app."

**Comment by tdruez** [comment]
@AvishrantsSh Well done. The report looks great 🚀 
Feel free to merge or to ask to @pombredanne to also have a look.

**Comment by AvishrantsSh** [comment]
@pombredanne can you have a look too?

**Pull request merged**